src/Services/ConvertToLearnosityService.php

```diff
@@ -96,16 +96,15 @@ public function process()
      * Performs a conversion on each directory (one level deep)
      * inside the given source directory.
      */
-    private function parseContentPackage()
-    {
+    private function parseContentPackage() {
```

Comment:
PSR-2: The opening brace should be on a newline for method declarations (looks good elsewhere though):
https://www.php-fig.org/psr/psr-2/#43-methods

Reply:
Done